templates/zerver/help/import-from-slack.md

```diff
@@ -221,100 +165,16 @@ token from being used to access your Slack workspace in the future.
-   and [how users can join your organization](/help/restrict-account-creation).
-
-2. All users from your Slack workspace will have accounts in your new Zulip
-   organization. However, you will need to let users know about their new
-   accounts, and [decide how they will log
-   in](/help/import-from-slack#decide-how-users-will-log-in) for the first time.
-
-3. Share the URL for your new Zulip organization, and (recommended) the [Getting
-   started with Zulip guide](/help/getting-started-with-zulip).
-
-4. Migrate any [integrations](/integrations), which is easy to do with Zulip's
-   [Slack-compatible incoming webhook](/integrations/doc/slack_incoming).
-
-{end_tabs}
-
-
-### Decide how users will log in
-
-When user accounts are imported from Slack, users initially do not have passwords
-configured. There are a few options for how users can log in for the first time.
+{!import-get-your-organization-started.md!}
 
 !!! tip ""
-    For security reasons, Slack passwords are never exported.
-
-### Allow users to log in with non-password authentication
-
-When you create your organization, users will immediately be able to log in with
-[authentication methods](/help/configure-authentication-methods) that do not
-require a password. Zulip offers a variety of authentication methods, including
-Google, GitHub, GitLab, Apple, LDAP and [SAML](/help/saml-authentication).
-
-### Send password reset emails to all users
 
-You can send password reset emails to all users in your organization, which
-will allow them to set an initial password.
-
-If you imported your organization into Zulip Cloud, simply e-mail
-[support@zulip.com](mailto:support@zulip.com) to request this.
-
-!!! warn ""
-    To avoid confusion, first make sure that
-    the users in your organization are aware that their Slack account has been
-    moved to Zulip, and are expecting to receive a password reset email.
-
-#### Send password reset emails (self-hosted organization)
-
-{start_tabs}
-
-{tab|default-subdomain}
-
-1. To test the process, start by sending yourself a password reset email by
-   using the following command:
-
-     ```
-     ./manage.py send_password_reset_email -u username@example.com
-     ```
-
-1. When ready, send password reset emails to all users by
-   using the following command:
-
-     ```
-     ./manage.py send_password_reset_email -r '' --all-users
-     ```
-
-{tab|custom-subdomain}
-
-1. To test the process, start by sending yourself a password reset email by
-   using the following command:
-
-     ```
-     ./manage.py send_password_reset_email -u username@example.com
-     ```
-
-1. When ready, send password reset emails to all users by
-   using the following command:
-
-     ```
-     ./manage.py send_password_reset_email -r <subdomain> --all-users
-     ```
-
-{end_tabs}
+    Zulip's [Slack-compatible incoming webhook](/integrations/doc/slack_incoming)
```

Comment:
Hmm, I do think this was better in-line rather than as a separate tip.

Reply:
I was hoping to reuse the entire block of content in the other pages by leaving this extra bit of info for Slack separately, but leaving it in-line and repeating step 4 works for me too.

Reply:
Sounds good, let's do that -- thanks!